feature/chat/src/main/java/com/serranoie/app/feature/chat/ChatScreen.kt

```
@@ -219,31 +199,96 @@ fun ChannelNameBar(
+        // First try to parse as Long (milliseconds)
+        val timeValue = rawTimestamp.toLongOrNull()
+        val messageDate: java.util.Date = if (timeValue != null) {
+            java.util.Date(timeValue)
+        } else {
+            // Try to parse as ISO 8601 format (2025-08-08T11:53:55.6130846)
+            try {
+                val isoFormatter = java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
+                val localDateTime = java.time.LocalDateTime.parse(rawTimestamp, isoFormatter)
+                val instant = localDateTime.atZone(java.time.ZoneId.systemDefault()).toInstant()
+                java.util.Date.from(instant)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)